includes/CLI.php

```diff
@@ -253,3 +290,38 @@ public static function coupons( $args, $assoc_args ) {
 		),
 	),
 ) );
+
+WP_CLI::add_command( 'wc generate terms', array( 'WC\SmoothGenerator\CLI', 'terms' ), array(
+	'shortdesc' => 'Generate product categories.',
+	'synopsis'  => array(
+		array(
+			'name'        => 'taxonomy',
+			'type'        => 'positional',
+			'description' => 'The taxonomy to generate the terms for.',
+			'options'     => array( 'product_cat', 'product_tag' ),
+		),
+		array(
+			'name'        => 'amount',
+			'type'        => 'positional',
+			'description' => 'The number of terms to generate. Max value 100.',
+			'optional'    => true,
+			'default'     => 10,
+		),
+		array(
+			'name'        => 'max_depth',
```

Comment:
We should change this to `max-depth` for consistency with the other commands and shell command parameters generally.

Reply:
Updated: 7c32e21